vm_insnhelper.c

```diff
@@ -5846,6 +5846,33 @@ vm_ic_update(const rb_iseq_t *iseq, IC ic, VALUE val, const VALUE *reg_ep, const
     rb_rjit_constant_ic_update(iseq, ic, pos);
 }
 
+static VALUE
+vm_opt_getconstant_path(rb_execution_context_t *ec, rb_control_frame_t *const reg_cfp, IC ic)
+{
+    VALUE val;
+    const ID *segments = ic->segments;
+    struct iseq_inline_constant_cache_entry *ice = ic->entry;
+    if (ice && vm_ic_hit_p(ice, GET_EP())) {
+        val = ice->value;
+
+        VM_ASSERT(val == vm_get_ev_const_chain(ec, segments));
+    } else {
+        ruby_vm_constant_cache_misses++;
+        val = vm_get_ev_const_chain(ec, segments);
+        vm_ic_track_const_chain(GET_CFP(), ic, segments);
+        // Because leaf=false, we need to undo the PC increment to get the address to this instruction
```

Comment:
```suggestion
        // Undo the PC increment to get the address to this instruction
```
Since we always increment PC now

Reply:
:+1: https://github.com/ruby/ruby/pull/8623/commits/ef8d915bd22624cbf214776297e00bdbc668c8d4